variables.tf

```diff
@@ -34,6 +34,31 @@ variable "acl" {
   default     = "private"
 }
 
+variable "cors_allowed_headers" {
+  description = "Specifies which headers are allowed."
+  default     = ["Authorization"]
+}
+
+variable "cors_allowed_methods" {
+  description = "Specifies which methods are allowed. Can be GET, PUT, POST, DELETE or HEAD."
+  default     = ["GET"]
```

Comment:
Are `Authorization` & `GET` default values for when no `cors_rule` is specified?

Reply:
As discussed and tested, these defaults are required and a permissive cors rule will get applied to each bucket that gets created